packages/web/src/pages/sign-up-page/pages/SignUpPage.tsx

```diff
@@ -34,30 +59,151 @@ type SignUpEmailValues = {
 
   return (
     <>
-      <Formik initialValues={initialValues} onSubmit={handleSubmit}>
-        <Form>
-          <h1>{messages.header}</h1>
-          <TextField name='email' label={messages.emailLabel} />
-          <Button variant={ButtonType.PRIMARY} type='submit'>
-            {messages.signUp}
-          </Button>
-        </Form>
-      </Formik>
-      <Text>{messages.haveAccount}</Text>
-      <Link to={SIGN_IN_PAGE}>{messages.signIn}</Link>
+      <form onSubmit={handleSubmit}>
+        <Flex
+          direction='column'
+          alignItems='center'
+          ph='2xl'
+          pv='4xl'
+          gap='2xl'
+        >
+          <PreloadImage
+            src={audiusLogoColored}
+            className={styles.logo}
+            alt='Audius Colored Logo'
+          />
+          <Flex
+            direction='column'
+            gap='l'
+            alignItems='flex-start'
+            w='100%'
+            className={styles.leftAlignText}
+          >
+            <Text color='heading' size='l' variant='heading' tag='h1'>
+              {messages.title}
+            </Text>
+            <Text color='default' size='l' variant='body' tag='h2'>
+              {messages.subHeader}
+            </Text>
+          </Flex>
+          <Flex direction='column' gap='l' w='100%' alignItems='flex-start'>
+            <TextInput
+              name='email'
+              autoFocus
+              autoComplete='email'
+              label={messages.emailLabel}
+              error={!!errors.email && touched.email}
+              helperText={
+                errors.email && touched.email ? errors.email : undefined
+              }
+              className={styles.w100}
+              onChange={(e) => {
+                // Clear errors on user change (Formik doesn't have an out of the box way to do this for you)
+                setErrors({ email: undefined })
+                handleChange(e)
+              }}
+              onBlur={handleBlur}
+              value={values.email}
+              disabled={isLoading}
+            />
+            <Flex w='100%' alignItems='center' gap='s'>
+              <Divider className={styles.flex1} />
+              <Text variant='body' size='m' tag='p' color='subdued'>
+                {messages.dividerText}
+              </Text>
+              <Divider className={styles.flex1} />
+            </Flex>
+            <Flex gap='s' w='100%' direction='row'>
+              <SocialButton socialType='twitter' className={styles.flex1} />
+              <SocialButton socialType='instagram' className={styles.flex1} />
+              <SocialButton socialType='tiktok' className={styles.flex1} />
+            </Flex>
+          </Flex>
+          <Flex direction='column' gap='l' alignItems='flex-start' w='100%'>
+            <Button
+              variant={ButtonType.PRIMARY}
+              type='submit'
+              fullWidth
+              iconRight={IconArrowRight}
+              isLoading={isLoading}
```

Comment:
I'm fine with that pattern, but I guess I would ask is that the UX pattern we prefer? aka visually disabled form buttons until the form is valid?
Technically visually disabling is optional since clicking the button doesn't submit until errors are empty anyways, so I think its purely a UX thing?